jobs/bok-choy_coverage.rb

```diff
@@ -14,7 +14,7 @@
 
     # Retrieve the coverage info as JSON
 
-    coverage_uri = JENKINS_JOB_URL + "/SHARD=1,TEST_SUITE=bok-choy,label_exp=jenkins-worker/#{last_build_num}/cobertura/_default_/api/json?depth=2"
```

Comment:
You are right, this is basically a no-op. But I thought I'd stay on top of the changes so that, when we do get bok-choy back, it will be a shorter hop... So, for consistency's sake, I left this change in.